docs/proposals/resourcebind-life-cycle/resourcebinding-behavior.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Behavior Of ResourceBind When ProPagation Policy ' s ResourceSelector Change
```

Comment:
```suggestion
title: PropagationPolicy Deletion Behavior
```
I guess we can take this as the title. As a user who might not care how Karmada deals with RB and even Work, he cares about whether the resource previously propagated to the member cluster will be removed or not.